src/FloatingButton/FloatingButton.tsx

```diff
@@ -201,7 +208,7 @@ export default class FloatingButton extends React.PureComponent {
                   cssClass.BUTTON,
                   colorGroup && cssClass.propStyle(colorGroup),
                 )}
-                onClick={button.onClick}
+                onClick={this.additionalButtonHandler(button)}
```

Comment:
you'll want to pre-bind this with an arrow function.
```
onclick={() => additionalButtonHandler(button)}
```
For more info see https://www.freecodecamp.org/news/this-is-why-we-need-to-bind-event-handlers-in-class-components-in-react-f7ea1a6f93eb/